pkg/cmd/attestation/trustedroot/trustedroot_test.go

```diff
@@ -85,11 +85,6 @@ func TestGetTrustedRoot(t *testing.T) {
 		TufRootPath: root,
 	}
 
-	t.Run("successfully verifies TUF root", func(t *testing.T) {
```

Comment:
I added a line in the testing GHA workflow that will run tests with the `integration` tag if you're interested in keeping online only tests for this command in the future.